lib/currency-for.js

```diff
@@ -0,0 +1,9 @@
+var currencies = require('../data/currencies.js');
+module.exports = {
+  stripe: Object.keys(currencies),
+  paypal: Object.keys(currencies).filter(function(i) {
+    if (!currencies[i].hasOwnProperty('disabled')) {
```

Comment:
Could this be simplified to `return !currencies[i].hasOwnProperty('disabled')` ?